index.js

```diff
@@ -31,6 +31,7 @@ class Ora {
 		this.stream = this.options.stream;
 		this.id = null;
 		this.frameIndex = 0;
+		this.indent = this.options.indent || 0;
```

Comment:
You now added an option, so should be documented as such, but I don't really see the use-case of starting with an indented spinner. Do you?